src/main/java/it/gov/pagopa/ranking/dto/mapper/OnboardingRankingRequest2EvaluationMapper.java

```diff
@@ -31,6 +31,7 @@ public EvaluationRankingDTO apply(OnboardingRankingRequests onboardingRankingReq
         evaluationRankingDTO.setOrganizationId(onboardingRankingRequests.getOrganizationId());
         evaluationRankingDTO.setBeneficiaryBudget(initiative.getBeneficiaryInitiativeBudget());
         evaluationRankingDTO.setOnboardingRejectionReasons(this.buildRejectionReasons(onboardingRankingRequests.getBeneficiaryRankingStatus(), onboardingRankingRequests.getRank()));
+        evaluationRankingDTO.setOrganizationName(initiative.getOrganizationName());
```

Comment:
let's keep fields ordered in order to find them more easily, put this after organizationId field